prime-router/src/main/kotlin/Element.kt

```diff
@@ -748,11 +741,11 @@ data class Element(
                 }
             }
             Type.TELEPHONE -> {
-                val number = phoneNumberUtil.parse(cleanedFormattedValue, "US")
-                if (!number.hasNationalNumber() || number.nationalNumber > 9999999999L)
-                    error("Invalid phone number '$cleanedFormattedValue' for $fieldMapping")
-                val nationalNumber = DecimalFormat("0000000000").format(number.nationalNumber)
-                "${nationalNumber}$phoneDelimiter${number.countryCode}$phoneDelimiter${number.extension}"
+                val (region, number) = isValidatePhoneNumber(cleanedFormattedValue)
```

Comment:
@oslynn please change the function name to `parsePhoneNumber` instead because this is what it's doing.

Reply:
Done.